### ISSUE TITLE
older macmini needs small tweek in smc.c

### ISSUE THREAD

**Issue by heegard** [lavoiesl/osx-cpu-temp#14]
I have an older macmini (some details below).  The compiler wouldn't work without this small change:

```
215c215,216
<     for (int i = 0; i < 10; i++) {
---
> 	int i;
>     for (i = 0; i < 10; i++) {
231c232,233
< 	for (int i = 0; i < nfans; i++) {
---
> 	int i;
> 	for (i = 0; i < nfans; i++) {

```
I guess the compiler doesn't allow declarations in the "for loop".  This small change allows it to compile and run.

-Chris

    Hardware Overview:

      Model Name: Mac mini
      Model Identifier: Macmini1,1
      Processor Name: Intel Core Duo
      Processor Speed: 1.66 GHz
      Number Of Processors: 1
      Total Number Of Cores: 2
      L2 Cache: 2 MB
      Memory: 2 GB
      Bus Speed: 667 MHz
  
Software:

    System Software Overview:

      System Version: Mac OS X 10.6.8 (10K549)
      Kernel Version: Darwin 10.8.0

**Comment by lavoiesl** [comment]
Can you submit a PR?

**Comment by lavoiesl** [comment]
Fixed by #15
